addons/default/visiosoft/advs-module/src/Http/Controller/advsController.php

```diff
@@ -151,29 +151,41 @@ public function index($category = null, $city = null)
         $isActiveDopings = $this->adv_model->is_enabled('dopings');
 
         // Search by category slug
+        $categoryId = null;
         if ($category) {
```

Comment:
You have checked the $category multiple time. Please Re-organize your code

Reply:
I checked it twice, once for the category section and once for the city section. I wanted to separate the category code from the city code @ozcan-durak